doc/plugins/eu.esdihumboldt.hale.doc.user/html/tasks/data/save_transformation.html

```diff
@@ -8,7 +8,7 @@
 	<h1>Save the transformation result</h1>
 	<p>
 		If you have defined a mapping and loaded source data in the
-		application, <i>hale�studio</i> performs the transformation derived from the
+		application, <i>hale�studio</i> performs the transformation derived from the
```

Comment:
The special character (or its encoding) between "hale" and "studio" seems to have been replaced with the changes done. Any chance to check if it will still display correctly in the online help?

Reply:
@stempler 

I can see the encoding issue in both the original version and in the new changes:

![grafik](https://github.com/halestudio/hale/assets/36956461/ba5248d7-edd3-4891-a69c-d0fcd149540e)

Is this expected? Is it a browser or GitHub display issue?

Reply:
To me it shows correct in GitHub for the old version:

![image](https://github.com/halestudio/hale/assets/375475/2751ddce-6404-4578-9766-1738a40f23bb)

Since currently the encoding is correct on the website it would probably be safest to leave it as it was before.

Reply:
@stempler If you refresh the page, you'll notice that both the `conversation `and `file changed` will be displayed with incorrect encoding.

Reply:
For me there is no difference when refreshing.
It may be shown correctly with the change, but we don't really know (maybe the new encoding is ISO-8859-1 as stated in the HTML file, but maybe that is actually not respected for the final help).
So just saying, since it's working right now in the final product, it would be the safe bet to not change it.